data-otservbr-global/scripts/movements/quests/hero_of_rathleton/professor_maxxen.lua

```diff
@@ -1,24 +1,26 @@
 local function startWaves()
-	local position = math.random(33704, 33718), math.random(32040, 32053), 15
+	local posX = math.random(33704, 33718)
+	local posY = math.random(32040, 32053)
+	local position = Position(posX, posY, 15)
```

Comment:
If think this is better:
```suggestion
	local position = Position(math.random(33704, 33718), math.random(32040, 32053), 15)
```